### DISCUSSION TITLE
Gas price on gas enabled network

### DISCUSSION THREAD

**Discussion by nicholaswhittle** [ConsenSys/quorum#1653] [Q&A]
On a gas enabled network: Is the gas price fixed? Or is it variable with network demand?

And what happens if stakeholders who run validators decide to change the gas price in the config? Will they not be able to sync / sign blocks? Or will it halt/break the chain?

If breaking happens, is there recommended practice to restrict all validators to use a shared common gas price?

**Comment by antonydenyer** [comment]
You can configure the rewards using:

https://docs.goquorum.consensys.net/configure-and-manage/configure/consensus-protocols/qbft#rewards

If nodes have different values it will create a fork and everything that goes with it!
